docs/get-started/testnet.md

```diff
@@ -14,28 +14,21 @@ After IRIS Hub 1.0 upgrade of mainnet, **Nyancat Testnet** starts to operate as
 
 ## Running a Node
```

Comment:
## Run a Full Node

### Start node from genesis

::tip You must use Irishub [v1.1.1](https://github.com/irisnet/irishub/releases/tag/v1.1.1)[ ](https://github.com/irisnet/irishub/releases/tag/v1.0.1) to initialize your node::